content/en/examples/examples_test.go

```diff
@@ -600,6 +600,12 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"node-problem-detector-configmap": {&apps.DaemonSet{}},
 			"termination":                     {&api.Pod{}},
 		},
+		"dra": {
+			"deviceclass":	{&resource.DeviceClass{}},
+      "resourceclaim": {$resource.ResourceClaim{}},
+      "resourceclaimtemplate": {$resource.ResourceClaimTemplate{}},
+      "dra-example-job": {$batch.Job{}},
```

Comment:
```suggestion
      "dra-example-job": {&batch.Job{}},
```

Reply:
how the heck did I confuse the two symbols lol thanks for catching it